js/models/conversations.js

```diff
@@ -142,8 +142,7 @@
                 || (beginningVerified !== verified && verified !== UNVERIFIED)
                 || (keychange && verified === VERIFIED)) {
 
-                var local = !options.viaSyncMessage && !options.viaContactSync;
-                this.addVerifiedChange(this.id, verified === VERIFIED, {local: local});
```

Comment:
Looks like this ensures that changes in state caused by contact syncs are not annotated as "from another device"... Android has no equivalent case for this choice since it never receives contact syncs. What's your reasoning here? Certainly if we got the change from a sync message, it didn't occur on this device.